backend/src/main/resources/data.sql

```diff
@@ -1,47 +1,26 @@
--- CLEAR
-
-UPDATE SURVEY -- REMOVE IT BECAUSE WE WILL RUN INTO CONSTRAINT PROBLEMS OTHERWISE
-SET CHOSEN_TIMESLOT_ID=NULL
-WHERE ID = 6;
-
-DELETE
-FROM RESPONSE_TIMESLOT
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM RESPONSE
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM TIMESLOT
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM PARTICIPATION
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM SURVEY
-WHERE ID <= 26
-  AND ID >= -1;
-DELETE
-FROM USER
-WHERE ID <= 26
-  AND ID >= -1;
```

Comment:
Have you tested this to still work in test and demo mode without emptying the database? Do you have an alternative?
Won't we run into problems with existing entities on the second server start?

Reply:
I removed that, because this will fail as soon as someone makes any changes to the db (add response, create survey, update survey, close survey) etc.